src/Logger-PREVAC_TMC13/dataframe.py

```diff
@@ -0,0 +1,143 @@
+import struct
```

Comment:
Because both libraries are rather short and are not used by other drivers so far (maybe TM1x?) it might be easier to move again into the main.py and thus resolve any potential future import problem.
Also general names like dataframe might make problems with other libraries that have a similar generic name. Maybe use tmc13_dataframe